### PR TITLE
Including HTTP response action in Docs

### PR DESCRIPTION
**Pull request by GTFalcao**
This is a complement for #3331 , to keep the docs up-to-date

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/3362"></a>

### CODE REVIEW

**Comment by dannyroosevelt** [comment]
tagging @dylburger and @ctrlaltdylan to review the docs changes

File: docs/docs/workflows/steps/triggers/README.md [hunk 1]
```diff
@@ -270,7 +270,17 @@ When you're processing HTTP requests, you often don't need to issue any special
 
 #### Customizing the HTTP response
 
-If you need to issue a custom HTTP response from a workflow, **you can use the `$.respond()` function in a Code or Action step**.
+If you need to issue a custom HTTP response from a workflow, you can either:
+- **use the `Return HTTP response` Action**, available on the `HTTP / Webhook` app, or
+- **use the `$.respond()` function in a Code or Action step**.
```
Comment (dylburger) [inline docs/docs/workflows/steps/triggers/README.md:275]: Capitalize "use"

File: docs/docs/workflows/steps/triggers/README.md [hunk 2]
```diff
@@ -270,7 +270,17 @@ When you're processing HTTP requests, you often don't need to issue any special
 
 #### Customizing the HTTP response
 
-If you need to issue a custom HTTP response from a workflow, **you can use the `$.respond()` function in a Code or Action step**.
+If you need to issue a custom HTTP response from a workflow, you can either:
+- **use the `Return HTTP response` Action**, available on the `HTTP / Webhook` app, or
```
[the remark this answers is not in the document]
Comment (GTFalcao) [reply docs/docs/workflows/steps/triggers/README.md:274]: Thanks, changed to bold

File: docs/docs/workflows/steps/triggers/README.md (hunk 1 above)
Comment (GTFalcao) [reply docs/docs/workflows/steps/triggers/README.md:275]: Capitalized both, thanks!

**Comment by GTFalcao** [comment]
Thanks for reviewing @dylburger , applied the requested changes.

Btw I think this should be merged only after PR #3360 is also merged ([currently awaiting QA validation](https://github.com/PipedreamHQ/pipedream/issues/3331))

**Comment by dylburger** [comment]
Awesome thanks. I'll go ahead and merge now since that should happen soon, that way we don't have to think about it later :)

**Pull request merged**